subt/docker/robotika/Dockerfile

```diff
@@ -8,6 +8,10 @@ RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_210405
 # example how to install an ubuntu package
 # RUN sudo apt-get update && sudo apt install -y ros-melodic-teleop-twist-keyboard
 
+RUN apt-get update \
+  && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends libnvidia-compute-460=460.32.03-0ubuntu1 \
```

Comment:
we should probably move this to base image if this will work on AWS, right?

Reply:
Yes, it would make sense.